src/libraries/System.Security.Cryptography/tests/X509Certificates/PfxTests.cs

```diff
@@ -431,7 +427,7 @@ public static void ReadMLKem512PrivateKey_Both_Pfx(X509KeyStorageFlags keyStorag
             byte[] pfxBytes = MLKemTestData.IetfMlKem512PrivateKeyBothPfx;
             string pfxPassword = MLKemTestData.EncryptedPrivateKeyPassword;
 
-            using (X509Certificate2 cert = X509CertificateLoader.LoadPkcs12(pfxBytes, pfxPassword, keyStorageFlags))
+            using (X509Certificate2 cert = X509CertificateLoader.LoadPkcs12(pfxBytes, pfxPassword, keyStorageFlags | X509KeyStorageFlags.Exportable))
```

Comment:
iOS, IIRC, doesn't support Exportable.

Since we don't know when we'll have any of these algorithms on iOS, this is probably a fine change; but at some point we might have to do something different to each of these.

Reply:
Changed the DSA algos to verify signature instead.